web/server/api/endpoints/files.py

```diff
@@ -65,12 +64,21 @@ async def write_file(
     else:
         (settings.project_path / path_or_new_path).write_text(content, encoding="utf-8")
 
+    found_models = [
+        model
+        for model in context.models.values()
+        if model._path == settings.project_path / path_or_new_path
+    ]
+    if found_models:
+        context.format_model(model=found_models[0])
+
+    path_or_new_path_mapping = await get_path_mapping(settings=get_settings())
```

Comment:
Settings wouldn't have changed, right?
```suggestion
    path_or_new_path_mapping = await get_path_mapping(settings=settings)
```